content/post/2024/02-sky130-kicad.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "sky130 SPICE, the KiCad way"
+date: 2024-02-12T19:09:55-0800
+description : "SPICE your ASIC schematics with KiCad!"
+tags: ["blog"]
+image: /static-2024/sky130-kicad/banner.png
+slug: sky130-kicad-spice
+draft: false
+---
+
+_This guest post is by [Aki Van Ness]._
+
+# sky130 SPICE, the KiCad way
+
+Picture this, you're working on a custom layout for [sky130], be it a PLL, or you're just trying your hands at making some basic building blocks. You've got a plan, and and ready to start laying down some nets, and that's when you must face it, the dreaded [xschem]!
+
+All joking aside, xschem is an incredibly powerful schematic capture system, but that power comes at a cost. xschem is fairly arcane to most, it has some graphical problems, and is just in general really hard to use if you're just starting out which makes rapid iteration difficult.
+
+Enter [KiCad], a robust and Open Source EDA software package, while it is mainly used for PCB layout, we can leverage the schematic capture and built-in SPICE simulation, right?
+
+## Introducing kicad-pdk-libs
+
+A little while ago I ran into [that exact problem](https://twitter.com/lethalbit/status/1498345181359321091), and so I set out and made [kicad-pdk-libs], a KiCad symbol library that adds schematic symbols for not only the basic building blocks of the sky130 PDK, but also SPICE library linkage.
```

Comment:
might be nice to have a little explainer of what PDK is? Feel free to link here: https://www.zerotoasiccourse.com/terminology/pdk/

Reply:
Maybe? I think knowing what PDK means is a semi-prerequisite to this, but I can try to work it in if you want.

It just felt a bit awkward trying to stick it in that section imo